docs/api/config.md

```diff
@@ -13,7 +13,7 @@ Configure jupyter notebook configuration
 
 **Arguments**:
 
-- `password` _str_ - Password for access authenticatioin
+- `password` _str_ - Password for access authentication
```

Comment:
Thanks for the fix!

The file is generated from https://github.com/tensorchord/envd/blob/main/envd/api/config/__init__.py#L49

Would you mind filing a new PR to fix there?

Reply:
later tonight